example-app/src/types.ts

```diff
@@ -13,3 +13,8 @@ export type IShortAccount = {
 };
 
 export type Api = IApi;
+
+export interface IPaymentMethodType {
```

Comment:
Q: Is there any benefit to use interface over type? 
Given type and interface are quite similar in typescript, if it's interchangeable here maybe we should pick the stricter one.

And also wondering if we want to promote the usage of `I` as prefix or not.
From my understanding it might not a good practice but open to discuss.
https://dev.to/mscamargo/why-you-should-avoid-using-the-i-prefix-for-interfaces-in-typescript-43gd

Reply:
removed!